### PR TITLE
Add migration for accounts

### PR DESCRIPTION
**Pull request by loan-laux**
Needed to run the migration introduced with [`api-plugin-accounts#17`](https://github.com/reactioncommerce/api-plugin-accounts/pull/17).

Merge when the next version of `api-plugin-accounts` gets released.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@reactioncommerce/api-plugin-accounts": "^1.3.0",
```
Comment (willopez) [inline package.json]: @loan-laux should this be updated to https://github.com/reactioncommerce/api-plugin-accounts/releases/tag/v1.4.0 now that the plugin has been published?

**Review by willopez — COMMENTED** [review]
The accounts plugins has been released...

File: package.json (hunk 1 above)
Comment (rawsp33d) [reply package.json]: Yeah we need v1.4.0. I'll update the PR.

Comment (rawsp33d) [reply package.json]: Oh wait I can't because it's not in our Github org. Duh!

**Comment by loan-laux** [comment]
There you go guys! Should be ready to merge now.

**Review by rosshadden — APPROVED** [review]

**Pull request merged**